Ssu7 (#571)
OctopusDeploy/blog: blog/2020-05/self-service-database-provisioning-with-octopus-runbooks-and-redgate-sql-clone/index.md

---

> **Edit, 22 Oct 2021:**

Reviewer: ```suggestion
**Edit - 22 Oct, 2021:**
```

> **Edit: 22nd Oct 2021**

Reviewer: ```suggestion
**Edit, 22 Oct 2021:**
```

> If you would like to learn more about database environment creation, as well as how it fits into broader DevOps strategy, you may enjoy my latest blog series. It starts [here](https://octopus.com/blog/safe-schema-updates-1-delivery-hell).

Reviewer: ```suggestion
If you would like to learn more about database environment creation, and how it fits into the broader DevOps strategy, you may enjoy my latest blog series about safe schema updates. It starts with [Database delivery hell](https://octopus.com/blog/safe-schema-updates-1-delivery-hell).
```
